lib/fetch/sanitize.js

```diff
@@ -164,19 +175,27 @@ export function sanitizeBadger(json) {
   const get = makeGetter(json);
   const name = get('badger.name');
   const sanitizeUrl = (url) => url && url.replace(/^http:/, 'https:');
-  const primaryImageUrl = pathOr(null, ['main', 'url'], get('badger.primary-image'));
-  const secondaryImageUrl = pathOr(null, ['main', 'url'], get('badger.secondary-image'));
   const fallbackPrimaryImageUrl = get('badger.image-url');
   const fallbackSecondaryImageUrl = get('badger.secondary-image-url');
+  const primaryImageUrl = pathOr(
+    sanitizeUrl(fallbackPrimaryImageUrl),
+    ['main', 'url'],
+    get('badger.primary-image')
+  );
+  const secondaryImageUrl = pathOr(
+    sanitizeUrl(fallbackSecondaryImageUrl),
+    ['main', 'url'],
+    get('badger.secondary-image')
+  );
   return {
     id: json.id,
     slug: json.uid,
     firstName: name && name.split(' ').slice(0, -1).join(' '),
     lastName: name && name.split(' ').slice(-1).join(' '),
     order: get('badger.order'),
     jobTitle: get('badger.job-title'),
-    primaryImageUrl: primaryImageUrl || sanitizeUrl(fallbackPrimaryImageUrl),
-    secondaryImageUrl: secondaryImageUrl || sanitizeUrl(fallbackSecondaryImageUrl),
+    primaryImageUrl,
```

Comment:
Why not put the `sanitizeUrl` here? Why do you only want to sanitize the fallback